Lib/test/test_tools/test_pathfix.py

```diff
@@ -17,15 +17,17 @@ def setUp(self):
         self.temp_file = support.TESTFN
         self.addCleanup(support.unlink, support.TESTFN)
 
-    def pathfix(self, shebang, pathfix_flags):
+    def pathfix(self, shebang, pathfix_flags, expected_returncode=0, stderr=''):
         with open(self.temp_file, 'w', encoding='utf8') as f:
             f.write(f'{shebang}\n' + 'print("Hello world")\n')
 
         proc = subprocess.run(
             [sys.executable, self.script,
              *pathfix_flags, '-n', self.temp_file],
             capture_output=True)
```

Comment:
please use text=1 instead of .decode() below, to use the proper encoding (locale encoding, rather than always using UTF-8).